lib/kafka_ex/gen_consumer.ex

```diff
@@ -698,7 +703,7 @@ defmodule KafkaEx.GenConsumer do
       end
 
     case response do
-      %{last_offset: nil, message_set: []} ->
```

Comment:
I'm not sure why we were matching on this before.  The new client passes the actual last offset even if the message set is empty.